app/metrics.yaml

```diff
@@ -2289,3 +2289,55 @@ cookie_banner:
     metadata:
       tags:
         - Privacy&Security
+
+  exception_added:
+    type: event
+    description: |
+      A user added a cookie banner handling exception through
+      the toggle in the protections panel.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1797578
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/
```

Comment:
Nit: Let's update the link after the approval :)

Reply:
Done